src/assets/sass/mixins/_utilits.mixins.scss

```
@@ -32,3 +36,128 @@
     opacity: 0.3;
     pointer-events: none;
 }
+
+// Yellow-blue text highlight mixins
+$line-height-560: 54px;
+$line-height-768: 76px;
+$line-height-1440: 116px;
+
+@mixin yellow-blue-text {
+    @include type.h3-medium-uppercase;
+    text-align: center;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)